components/ProjectList/ProjectList.js

```diff
@@ -0,0 +1,38 @@
+import styled from "styled-components";
+import Image from "next/image";
+import React from "react";
+import { projects } from "../../utils/data";
+
+const ProjectPreview = styled.div`
+  background-color: #faf8f7;
+  margin-left: 20px;
+  padding: 0 10px 10px 10px;
+  margin-bottom: 20px;
+  border-radius: 5%;
+`;
+
+const StyledImage = styled(Image)`
+  height: 100%;
+  width: 100%;
+  border-radius: 5%;
+`;
+
+export default function ProjectList() {
+  return (
+    <div>
+      {projects.map((project) => (
+        <ProjectPreview key={project.id} className={`div__${project.category}`}>
```

Comment:
does the `className` here have a use? also ideally don't use classes for the Capstone and instead only apply `styled-components` to keep things simple and clear

Reply:
Thanks for your feedback. I would like to style the divs differently regarding their category but I just figured out that I also can use conditionals in a styled component.